packages/node/src/integrations/tracing/prisma.ts

```diff
@@ -13,6 +13,19 @@ const _prismaIntegration = (() => {
         new prismaInstrumentation.PrismaInstrumentation({}),
       );
     },
+
+    setup(client) {
+      client.on('spanStart', span => {
```

Comment:
Sadly we have no hook in the integration for this, but I think this is OK - as this integration is opt-in anyhow (as of now, at least), there is no overhead if not using this.

Reply:
Possible improvement for this: We could PR this upstream to have a proper hook for this...